plugins/helphub-post-types/assets/css/admin.css

```diff
@@ -1,9 +1,612 @@
 .post-type.edit-php table.wp-list-table .column-image {
-	width: 120px;
-	text-align: center;
+  width: 120px;
+  text-align: center; 
 }
 
 .post-type.edit-php table.wp-list-table .column-image img {
 	width: 60px;
-	height: auto;
+	height: auto; 
 }
+
+
```

Comment:
Below this line comment is the addition of jQuery UI v1.12.1, is the plan to update this via future jQuery UI versions?

If yes, it should be it's own file and include it via `@include`

If not a follow up PR for CSS coding standards ( Would be easier in a follow up PR rather than this PR)